lgc/patch/VertexFetch.cpp

```diff
@@ -1849,8 +1849,10 @@ bool VertexFetchImpl::needPatchA2S(const VertexInputDescription *inputDesc) cons
 
   if (inputDesc->dfmt == BufDataFormat2_10_10_10 || inputDesc->dfmt == BufDataFormat2_10_10_10_Bgra) {
     if (inputDesc->nfmt == BufNumFormatSnorm || inputDesc->nfmt == BufNumFormatSscaled ||
-        inputDesc->nfmt == BufNumFormatSint)
-      needPatch = m_lgcContext->getTargetInfo().getGfxIpVersion().major < 9;
+        inputDesc->nfmt == BufNumFormatSint) {
+      needPatch = true;
```

Comment:
I think this should be `needPatch = false`.

Reply:
Thanks. Will fix.